Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -211,4 +204,11 @@ def git_log_trailers_cmd(repo_path, person, trailer, args)
 
     Utils.safe_popen_read(*cmd).lines.count { |l| l.include?(person) }
   end
+
+  sig { params(repo_full_name: String, person: String, args: Homebrew::CLI::Args).returns(Integer) }
+  def count_reviews(repo_full_name, person, args)
+    GitHub.count_issues("", is: "pr", repo: repo_full_name, reviewed_by: person, review: "approved", args: args)
```

Comment:
Just for future reference: I'd like us to avoid passing `args` beyond the command file's boundary.

Reply:
Actually, it seem's `args` is not used by `count_issues` at all anyways.

Reply:
> I'd like us to avoid passing args beyond the command file's boundary.

Since we do this elsewhere in this file too, I'll fix all these in a separate PR.

> Actually, it seem's `args` is not used by `count_issues` at all anyways.

It is, it's just confusing since it's not directly referenced until several method calls away when we construct the search query string: https://github.com/github/turboscan/blob/af68a976fb1f866b65b7e224ecf99cbb488dcd10/ts/mysql/processor.go#L1045.

Reply:
Wow that is way too deep down the call chain.